lib/workers/repository/finalize/prune.ts

```diff
@@ -113,7 +113,7 @@ export async function pruneStaleBranches(
   let renovateBranches = getBranchList().filter(
     (branchName) =>
       branchName.startsWith(config.branchPrefix!) &&
-      branchName !== getReconfgiureBranchName(config.branchPrefix!)
+      branchName !== getReconfgiureBranchName(config.branchPrefix)
```

Comment:
typo in function name 🙈 
```suggestion
      branchName !== getReconfigureBranchName(config.branchPrefix)
```